src/hub/mod.rs

```diff
@@ -22,8 +24,22 @@ use crate::{
 use actor::HubActor;
 use protocol::HubCommand;
 
+lazy_static! {
+    static ref HUB: AsyncOnce<Hub> = AsyncOnce::new(async {
+        Hub::new(
+            10000,
+            Arc::new(RwLock::new(
+                mavlink::ardupilotmega::MavComponent::MAV_COMP_ID_ONBOARD_COMPUTER as u8,
+            )),
+            Arc::new(RwLock::new(1)),
+            Arc::new(RwLock::new(1.)),
+        )
+        .await
+    });
+}
+
 #[derive(Clone)]
-pub struct Hub {
+struct Hub {
```

Comment:
If we are exposing this via hub::clone(), then it must be public

```suggestion
pub struct Hub {
```

Reply:
nice.